src/vm_cpu.rs

```diff
@@ -296,6 +330,26 @@ pub struct ActiveVmCpu<'vcpu, 'pages, 'prev, T: GuestStagePagingMode> {
     parent_vcpu: Option<&'prev mut dyn VmCpuSaveState>,
 }
 
+#[cfg(target_feature = "v")]
+fn restore_vector(state: *mut VmCpuState) {
+    unsafe {
+        // safe because the only memory it touches is known offsets of _state
```

Comment:
Nit: Capitalize first letter here, and elsewhere in other files.

Reply:
fixed